crt_portal/cts_forms/templates/forms/question_cards/single_question.html

```diff
@@ -1,4 +1,5 @@
-<div class="form-group">
+<div class="form-group"{% if hidden == True %} style="display:none" id="div-{{ field.id_for_label }}">
```

Comment:
Opportunity to trim this to only evaluate the truthiness of `hidden`

```liquid
{% if hidden %}
```

Reply:
That looks good, I will make that now. 

Also there is a new github feature where you can use:
![Screen Shot 2020-04-21 at 1 45 22 PM](https://user-images.githubusercontent.com/4406333/79912203-807ae880-83d6-11ea-9f3b-63ea5acf0917.png)
to make a suggestion. I find it good for these use cases. If you don't feel like using that, no worries.